epub33/rs/index.html

```diff
@@ -973,9 +998,28 @@ <h4>The <code>rendition:orientation</code> Property</h4>
 					<p>The default global value is <code>auto</code> if no <code>meta</code> element carrying this
 						property occurs in the <code>metadata</code> section.</p>
 
-					<p>Reading Systems that support multiple orientations SHOULD convey the intended orientation to the
-						user unless the given value is <code>auto</code>. The means by which they convey the intent is
-						implementation specific.</p>
+					<p>The <code>rendition:layout</code> property values have the following processing requirements:</p>
+
+					<dl class="variablelist">
+						<dt id="def-orientation-auto">auto</dt>
+						<dd>
+							<p>The Reading System determines the orientation in which to render the content.</p>
+						</dd>
+
+						<dt id="def-orientation-landscape">landscape</dt>
+						<dd>
+							<p>Reading Systems that support multiple orientations SHOULD render the content in lanscape
+								orientation.</p>
+						</dd>
+
+						<dt id="def-orientation-portrait">portrait</dt>
+						<dd>
+							<p>Reading Systems that support multiple orientations SHOULD render the content in portrait
+								orientation.</p>
+						</dd>
+					</dl>
+
+					<p>The means by which they convey the intent is implementation specific.</p>
```

Comment:
Does anyone want to hazard a guess at what this sentence means? "convey the intent" sounds like relaying on to the user what the author is asking for in the metadata, but that makes no sense.

I would assume it is trying to say how the reading system puts the content in the desired orientation is implementation specific, but how is that not just stating the obvious?

I'd like to delete this sentence, but want to make sure there isn't some hidden meaning I'm not grasping here.

Reply:
The original sentence was

> Reading Systems that support multiple orientations should, unless the given value is 'auto', convey the intended orientation to the user. The means by which the intent is conveyed is implementation-specific.

Ah. I wonder if we were imagining that FXL reading systems would have user controls to choose orientation. And so the reading system would need some way to show the user which orientation that the content creator wanted. How to do that would be decidedly implementation-specific, as it is about the user interface of the reading system.

Reply:
> I wonder if we were imagining that FXL reading systems would have user controls to choose orientation.

But that's what's weird about this. The first sentence is the requirement to render it in the given orientation, or so I can only assume. "convey the intended orientation to the user" sounds like Swedish for "display in the specified orientation to the user", if I were to guess. 😉 

It always could be an artefact of an early draft of the FXL document for all I know, but why would only this property define such a thing?

In any case, since we don't define user control over FXL I assume that's also a case for dropping it.